test/linters/button_component_migration_counter_test.rb

```diff
@@ -99,6 +99,22 @@ def test_autocorrects
     assert_equal expected, corrected_content
   end
 
+  def test_autocorrects_attributes
+    @file = <<~HTML
+      <button class="btn" href="href" value="value" name="name" tabindex="tabindex">
```

Comment:
Is a button with an `href` valid? I thought we tried to avoid that combination.

Reply:
buttons can be `<a>`, so they have `hrefs`